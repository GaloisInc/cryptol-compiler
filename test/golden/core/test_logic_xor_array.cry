main = [0xF0, 0x10] ^ [0xD1, 0xFF]