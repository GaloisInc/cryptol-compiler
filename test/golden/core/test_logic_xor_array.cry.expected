[0x21, 0xef]
